### DISCUSSION TITLE
Better UK Overview page ~ I may have a Solution.

### DISCUSSION THREAD

**Discussion by NOOBLINUXUSER** [MY-Sabil/beyondborders-wiki#5] [Ideas]
I was researching on UK and Stumbled across Kazi Tasnim Ferdous's Overview on Facebook. This was before I found out about the wiki page you guys have created - I might started my research before the wiki was even announced - and I for my own sanity used markdown to better organize the post and added flairs like headings and bullet points so I can better digest the informations and also Have an overview of the things I am working on/need to know. 

Here is what It looks like in Obsidian - this is my personal vault full of personal stuff like you would have in your personal dairy. 
![image](https://github.com/MY-Sabil/beyondborders-wiki/assets/66922885/3de161c9-84f8-431c-9206-fb842f128f20) and I also want to do a pull request by providing the mark down (all of my files are markdown and uses that syntax) if that is okay with the Up-stream. I am new to all of this, so I didn't want you guys to have your first pull request be from someone who is barely new to github and all its intricacy.

**Comment by MY-Sabil** [comment]
Sure!! Just create pull requests for the pages you want to add and if it needs any changes, we can figure it out later.

Thank you so much for contributing to the wiki. If you need any help with github or anything, just reply to this thread or send me a message directly on any of my socials. 😀

**Reply by NOOBLINUXUSER** [reply]
sorry bro, i was swamped with errands and choirs and basically just busy. I will upload it asap - roughly about 2 to 3 days from now, might also add a few more better wiki pages. I do know a bit of Html and CSS (learning JS - just started) and did 101 in python. So if there is anything technical i need to do as well for a proper pull request, could you help me? Lets sit together for 10 mins (max i think, i been using github for backuping obsidian docs), would be faster.

**Comment by MY-Sabil** [comment]
Yeah sure. You can schedule a meeting from here:
https://calendar.notion.so/meet/mysabil4/im4u4e1s.

**Reply by NOOBLINUXUSER** [reply]
I can do it tomorrow, if you are willing to. The link expired so I couldn't reschedule.

**Reply by NOOBLINUXUSER** [reply]
From what I understand, I need to first fork it. Then edit (replacing) the file in my local directory and, then commit and push. Isn't that right?

**Comment by MY-Sabil** [comment]
Sure, I can do it tomorrow. Here's an updated link:
https://calendar.notion.so/meet/mysabil4/kjo94ea3. Schedule the meeting
whenever you are free.
